public/controllers/management/components/management/management-main.js

```diff
@@ -77,7 +78,15 @@ class WzManagementMain extends Component {
           (section === 'statistics' && <WzStatistics />) ||
           (section === 'logs' && <WzLogs />) ||
           (section === 'configuration' && <WzConfiguration {...this.props.configurationProps} />) ||
-          (ruleset.includes(section) && (
+          (section === 'decoders' && <WzDecoders
+            logtestProps={this.props.logtestProps}
+            clusterStatus={this.props.clusterStatus}
+          />) ||
+          (section === 'lists' && <WzCDBLists
+            logtestProps={this.props.logtestProps}
+            clusterStatus={this.props.clusterStatus}
+          />) ||
+          (['ruleset', 'rules'].includes(section) && (
```

Comment:
Constants (SECTION_CDBLIST_SECTION, ...) have been added, but are not being used here.